app/src/test/java/memphis/myapplication/TLVEncodeDecodeTest.java

```diff
@@ -1,35 +0,0 @@
-package memphis.myapplication;
-
-import android.app.Activity;
-
-import net.named_data.jndn.Interest;
-import net.named_data.jndn.Name;
-import net.named_data.jndn.util.Blob;
-
-import org.junit.Test;
-
-import java.util.ArrayList;
-import java.util.Arrays;
-
-import javax.crypto.SecretKey;
-
-import memphis.myapplication.UI.MainFragment;
-import memphis.myapplication.psync.ConsumerManager;
-import memphis.myapplication.data.tasks.FetchingTaskParams;
-
-import static org.junit.Assert.assertThat;
-
-public class TLVEncodeDecodeTest {
```

Comment:
Why is this deleted?

Reply:
We switched to JSON for the file metadata over the summer and aren't using TLV anymore.